core/src/main/scala/org/apache/spark/deploy/worker/DriverRunner.scala

```diff
@@ -57,7 +57,8 @@ private[deploy] class DriverRunner(
   @volatile private[worker] var finalException: Option[Exception] = None
 
   // Timeout to wait for when trying to terminate a driver.
-  private val DRIVER_TERMINATE_TIMEOUT_MS = 10 * 1000
+  private val DRIVER_TERMINATE_TIMEOUT_MS =
+    conf.getTimeAsMs("spark.worker.driverTerminateTimeout", "10s")
```

Comment:
`spark.worker` means this is only for Spark workers, so I think it should be obvious. Do you have a better config name?